app/src/main/res/layout/swipe_illustration_content.xml

```diff
@@ -47,14 +47,17 @@
 
     <com.google.android.material.imageview.ShapeableImageView
         android:id="@+id/description1"
-        android:layout_width="236dp"
+        android:layout_width="0dp"
         android:layout_height="7dp"
         android:layout_marginStart="@dimen/marginStandard"
         android:layout_marginTop="12dp"
         android:layout_marginBottom="@dimen/marginStandardSmall"
         android:background="#C4C4C4"
```

Comment:
Hardcoded color in a layout, is it on purpose?

Reply:
Yes, I recreated the illustration to have created control on how it's displayed across multiple width of screen. But these color don't really represent anything else in the app